### PR TITLE
Updates for 2023 Nemo training

### PR DESCRIPTION
**Pull request by arporter**

### CODE REVIEW

**Comment by arporter** [comment]
Some minor updates to the NEMO-related tutorials and Jupyter notebooks. One for anyone to review :-)

**Comment by arporter** [comment]
Apart from discovering #1006 (see comment), the tutorials all work as expected. I've improved the text in a few places though.
This really is ready for review now.

File: tutorial/practicals/nemo/3_nemo_openmp/README.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ the number of MPI processes and resulting inter-process communication.)
    directives have been inserted. You should see that there are now
    `Directive` nodes in the PSyIR, e.g.:
 
-       14: OMPParallelDoDirective[]
+       14: OMPParallelDoDirective[omp_schedule=auto]
```
Comment (rupertford) [inline tutorial/practicals/nemo/3_nemo_openmp/README.md:145]: The nemo part of the notebooks states the schedule is "static" not "auto". Could you please change this.

**Review by rupertford — CHANGES_REQUESTED** [review]
The changes look good. I've not had time to run through the tutorial itself but seeing as you've just changed the README descriptions I'm assuming all is still OK. I did go through the notebooks and they  worked fine for me. I just have a couple of requests. Please see inline comments.

File: tutorial/notebooks/fparser2/parsing_fortran.ipynb [hunk 2]
```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This example shows how PSyclone reads existing Fortran code using a parser called fparser2."
+    "This example shows how PSyclone reads existing Fortran code using a parser called fparser2 (https://pypi.org/project/fparser/)."
```
[the remark this answers is not in the document]
Comment (arporter) [reply tutorial/notebooks/fparser2/parsing_fortran.ipynb:14]: Oh yes, rationalised.

File: tutorial/practicals/nemo/3_nemo_openmp/README.md (hunk 1 above)
Comment (arporter) [reply tutorial/practicals/nemo/3_nemo_openmp/README.md:145]: Fixed the nemo notebook (well spotted).

**Comment by arporter** [comment]
Assuming CI happiness, this is ready for another look now.

**Pull request merged**